caffe2/python/onnx/backend_cpp_rep.py

```diff
@@ -33,6 +33,18 @@ def __init__(self, cpp_rep):
         self.__external_inputs = cpp_rep.external_inputs()
         self.__uninitialized_inputs = cpp_rep.uninitialized_inputs()
 
+    def init_net(self):
+        return self.__core.init_net()
```

Comment:
nit: you can use `def_property_readonly` to make `init_net` a property instead of a member function of `__core`.